src/app/components/game-components/game-component/GameComponent.ts

```diff
@@ -167,6 +167,9 @@ export abstract class GameComponent<R extends SuperRules<M, S, C, L>,
 
     public cancelMoveAttempt(): void {
         // Override if need be
+        // Only call it manually in the component if you want (by example) to switch the selected piece
```

Comment:
for example, puis pas mettre en parenthèses (sinon ça veut dire que tu peux le supprimer, et ici ça change la sémantique !)